src/Sentry/Laravel/Tracing/EventHandler.php

```diff
@@ -255,6 +275,45 @@ private function resolveQueryOriginFromBacktrace(): ?string
         return "{$filePath}:{$firstAppFrame->getLine()}";
     }
 
+    protected function redisCommandExecutedHandler(RedisEvents\CommandExecuted $event): void
+    {
+        if (!$this->traceRedisCommands) {
+            return;
+        }
+
+        $parentSpan = SentrySdk::getCurrentHub()->getSpan();
+
+        // If there is no tracing span active there is no need to handle the event
+        if ($parentSpan === null) {
+            return;
+        }
+
+        // @TODO: Do we need/want to prefix all data with `db.redis`? It's already in the `db.redis` op so maybe it's not needed?
```

Comment:
Can't find much guidance on what to do here exactly, do we prefix all data with the OP or no (or sometimes?)?

Reply:
You can keep it as is.